migration.go

```diff
@@ -28,6 +30,13 @@ func checkRepository() bool {
 	}
 	log.Info("Elementum repository not installed, installing...")
 	xbmc.InstallAddon("repository.elementum")
+	for timeout := 0; timeout < 3; timeout++ {
```

Comment:
Since it can be a slow or high-loaded device, 3 seconds can be not enough. 
Even 10 seconds is fine for this. Same for `config.go`.

Reply:
done.